### DISCUSSION TITLE
Server migration

### DISCUSSION THREAD

**Discussion by gigarun** [amidaware/tacticalrmm#1894] [Q&A]
Hi, i’ve followed [backup](https://docs.tacticalrmm.com/backup/) / [restore](https://docs.tacticalrmm.com/restore/) method to change my tactical RMM Vm to another Vm (pointing rmm.mydomain.com to new Vm with mesh.mydomain.com and api.mydomain.com DNS A to new ip address)
But if I poweroff old vm I can’t login to thenew one. 
I have also reseted 2Fa but still the same.
What am i missing ?

**Comment by dinger1986** [comment]
is your dns setup correctly? did the restore work properly?

**Comment by gigarun** [comment]
Yes, for the 2 questions.

En vous remerciant [Skype me<https://join.skype.com/invite/B0EfgOD999wy>] or [Teams ***@***.***>]
Appréciations : pagesjaunes.fr <https://www.pagesjaunes.fr/contribution/votre-avis/52616170?codeRubrique=00050220> ou Gmaps<https://g.page/r/CWsRzfTPHuDJEAg/review> ou ici<https://www.donnersonavis.com/listing/gigarun-transformation-numerique/>
Thomas CASSEN<https://1162.3cx.cloud/thomascassen> CEO | Consultant TIC & QSE|

[Gigarun ingénierie]<https://www.gigarun.re/>
***@***.***?subject=Contact>
***@***.******@***.***>
[url]<https://faceebook.com/gigaruningenierie>

[tel]
0262 967 291
[url]<https://twitter.com/gigarun>

[gsm]
0692 311 191
[url]<https://instagram.com/gigaruningenierie>

[maps]<https://goo.gl/maps/Yyhq3hfTC1t4cM7h6>
7 rue du temple
***@***.***>

[url]<https://www.gigarun.re/>
www.gigarun.re<http://www.gigarun.re>
[url]<https://www.linkedin.com/company/gigarun/>

[url]<https://tawk.to/chat/5b1e07e13604f81d726beaf9/default>
[url]<https://1162.3cx.cloud/callus/#gigaruningenierie>
[url]<https://wa.me/262692311191>
[url]<https://t.me/gigarun>
[url]<https://m.me/gigaruningenierie>
[url]<https://linkedin.com/in/thomascassen>

Support<https://support.gigarun.net/> 8-12h|14-18h
[url]<https://support.gigarun.net/>
Lundi - vendredi

Téléchargement de la<https://www.gigarun.re/images/pdf/Procedure_support.pdf>
[url]<https://www.gigarun.re/images/pdf/Procedure_support.pdf>
procédure support<https://www.gigarun.re/images/pdf/Procedure_support.pdf>

Prenons rendez-vous ensemble maintenant ***@***.***?anonymous&ep=plink>

De : dinger1986 ***@***.***>
Envoyé : lundi 17 juin 2024 16:48
À : amidaware/tacticalrmm ***@***.***>
Cc : Thomas Cassen ***@***.***>; Author ***@***.***>
Objet : Re: [amidaware/tacticalrmm] Server migration (Discussion #1894)


is your dns setup correctly? did the restore work properly?

**Comment by dinger1986** [comment]
has it updated on the machines? sounds like a dns issue

**Comment by gigarun** [comment]
Maybe i’d have to wait DNS propagation til tomorrow ?

En vous remerciant [Skype me<https://join.skype.com/invite/B0EfgOD999wy>] or [Teams ***@***.***>]
Appréciations : pagesjaunes.fr <https://www.pagesjaunes.fr/contribution/votre-avis/52616170?codeRubrique=00050220> ou Gmaps<https://g.page/r/CWsRzfTPHuDJEAg/review> ou ici<https://www.donnersonavis.com/listing/gigarun-transformation-numerique/>
Thomas CASSEN<https://1162.3cx.cloud/thomascassen> CEO | Consultant TIC & QSE|

[Gigarun ingénierie]<https://www.gigarun.re/>
***@***.***?subject=Contact>
***@***.******@***.***>
[url]<https://faceebook.com/gigaruningenierie>

[tel]
0262 967 291
[url]<https://twitter.com/gigarun>

[gsm]
0692 311 191
[url]<https://instagram.com/gigaruningenierie>

[maps]<https://goo.gl/maps/Yyhq3hfTC1t4cM7h6>
7 rue du temple
***@***.***>

[url]<https://www.gigarun.re/>
www.gigarun.re<http://www.gigarun.re>
[url]<https://www.linkedin.com/company/gigarun/>

[url]<https://tawk.to/chat/5b1e07e13604f81d726beaf9/default>
[url]<https://1162.3cx.cloud/callus/#gigaruningenierie>
[url]<https://wa.me/262692311191>
[url]<https://t.me/gigarun>
[url]<https://m.me/gigaruningenierie>
[url]<https://linkedin.com/in/thomascassen>

Support<https://support.gigarun.net/> 8-12h|14-18h
[url]<https://support.gigarun.net/>
Lundi - vendredi

Téléchargement de la<https://www.gigarun.re/images/pdf/Procedure_support.pdf>
[url]<https://www.gigarun.re/images/pdf/Procedure_support.pdf>
procédure support<https://www.gigarun.re/images/pdf/Procedure_support.pdf>

Prenons rendez-vous ensemble maintenant ***@***.***?anonymous&ep=plink>

De : dinger1986 ***@***.***>
Envoyé : lundi 17 juin 2024 17:04
À : amidaware/tacticalrmm ***@***.***>
Cc : Thomas Cassen ***@***.***>; Author ***@***.***>
Objet : Re: [amidaware/tacticalrmm] Server migration (Discussion #1894)


has it updated on the machines? sounds like a dns issue

**Reply by dinger1986** [reply]
depends on your dns host but I would give it time to propagate
